mantidproject/mantid
# Fix order of merging in MergeRuns

**Description of work.**

This PR removes the confusing sorting of input workspaces according to their X values from `MergeRuns`. Now the workspaces are handled in the order they are listed in the *InputWorkspace* property, as one would expect.

**Report to:** Björn Fåk.

**To test:**

Check that the `merged` workspace in the following has `ws_number == 2.` in the sample logs:

```python
Xs = [0., 1.]
Ys = [0.]
ws1 = CreateWorkspace(DataX=Xs, DataY=Ys)
ws1.mutableRun().addProperty('ws_number', 1., True)
Xs = [1., 2.]
ws2 = CreateWorkspace(DataX=Xs, DataY=Ys)
ws2.mutableRun().addProperty('ws_number', 2., True)
merged = MergeRuns(['ws2', 'ws1'])
```

Fixes #22402.

Does this update require release notes?

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
